gui/system/migrations/0022_cert_serial.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-05-14 16:14
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import freenasUI.freeadmin.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('system', '0021_merge_20180507_1315'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='certificate',
+            name='cert_serial',
+            field=models.CharField(blank=True, help_text='Serial for next certificate', max_length=120, null=True, verbose_name='Serial'),
```

Comment:
We probably want `48` for max_length, which is what 2^159-1 is

Reply:
I've changed the maximum to 48 in latest commit.